cephfs/path.go

```diff
@@ -44,3 +44,68 @@ func (mount *MountInfo) RemoveDir(path string) error {
 	ret := C.ceph_rmdir(mount.mount, cPath)
 	return getError(ret)
 }
+
+// Link creates a new link to an existing file
+//
+// Implements:
+// int ceph_link (struct ceph_mount_info *cmount, const char *existing,
```

Comment:
Add an additional space before 'int' to make this act more like a quoted block in the godoc output. This applies to the other new doc-comments in this patch.